From bf63ba75f327e5a50092c94f1cffa5e8c61de782 Mon Sep 17 00:00:00 2001
From: Lucas De Marchi <lucas.demarchi@intel.com>
Date: Fri, 21 Apr 2017 02:25:45 -0700
Subject: [PATCH 15/27] regulator: whiskey_cove: fixup to build with gcc >= 6

Signed-off-by: Lucas De Marchi <lucas.demarchi@intel.com>
---
 drivers/regulator/pmic_whiskey_cove.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/regulator/pmic_whiskey_cove.c b/drivers/regulator/pmic_whiskey_cove.c
index 0a26f82..2e9a53f 100644
--- a/drivers/regulator/pmic_whiskey_cove.c
+++ b/drivers/regulator/pmic_whiskey_cove.c
@@ -150,7 +150,7 @@ static unsigned int WCOVE_V3P3A_VSEL_TABLE[WCOVE_V3P3A_VRANGE],
  * The VSDIO regulator should only support 1.8V and 3.3V. All other
  * voltages are invalid for sd card, so disable them here.
  */
-static const unsigned int WCOVE_VSDIO_VSEL_TABLE[WCOVE_VSDIO_VRANGE] = {
+static unsigned int WCOVE_VSDIO_VSEL_TABLE[WCOVE_VSDIO_VRANGE] = {
 	0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
 	0, 0, 0, 0, 0, 0, 0, 0, 1800000, 0, 0, 0,
 	0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-- 
2.9.3

